frappe/public/js/legacy/clientscriptAPI.js

```diff
@@ -158,11 +169,18 @@ _f.Frm.prototype.get_docfield = function(fieldname1, fieldname2) {
 	}
 }
 
-_f.Frm.prototype.set_df_property = function(fieldname, property, value) {
-	var field = this.get_docfield(fieldname);
+_f.Frm.prototype.set_df_property = function(fieldname, property, value, docname, table_field) {
```

Comment:
Hi Max. Same as above. Why is this code necessary? Comments in the code are welcome :)